packages/react-composites/src/composites/CallComposite/components/Prompt.tsx

```diff
@@ -49,9 +49,9 @@ export const Prompt = (props: PromptProps): JSX.Element => {
       <Stack verticalAlign="center" className={mergeStyles({ minHeight: '6rem' })}>
         <Text className={mergeStyles({ fontSize: '0.875rem' })}>{props.text}</Text>
       </Stack>
-      <Stack horizontal reversed tokens={{ childrenGap: '0.5rem' }}>
-        <DefaultButton styles={buttonTextStyles} text={props.cancelButtonLabel} onClick={props.onCancel} />
+      <Stack horizontal horizontalAlign="end" tokens={{ childrenGap: '0.5rem' }}>
         <PrimaryButton styles={buttonTextStyles} text={props.confirmButtonLabel} onClick={props.onConfirm} />
+        <DefaultButton styles={buttonTextStyles} text={props.cancelButtonLabel} onClick={props.onCancel} />
```

Comment:
This change is to fix tab order so that the order is:
![image](https://github.com/Azure/communication-ui-library/assets/79475487/d48beb3e-41b2-4138-b382-161bee224953)